rdf4j/src/test/java/com/inrupt/client/rdf4j/RDF4JBodyHandlersTest.java

```diff
@@ -61,14 +63,14 @@ static void teardown() {
     }
 
     @Test
-    void testOfModelHandlerAsync() throws IOException,
+    void testOfRDF4JModelHandlerAsync() throws IOException,
             InterruptedException, ExecutionException, TimeoutException {
         final Request request = Request.newBuilder()
                 .uri(URI.create(config.get("rdf_uri") + "/oneTriple"))
                 .GET()
                 .build();
 
-        final Response<Model> res = client.send(request, RDF4JBodyHandlers.ofModel()).toCompletableFuture().join();
+        final Response<Model> res = client.send(request, RDF4JBodyHandlers.ofRDF4JModel()).toCompletableFuture().join();
```

Comment:
When a method is deprecated should we retain the tests for it and remove them only when the deprecated methods are removed?

Reply:
You're right, the deprecated method should be covered until removed.